content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -403,6 +410,27 @@ In the past, the annotation `volume.beta.kubernetes.io/mount-options` was used i
 of the `mountOptions` attribute. This annotation is still working, however
 it will become fully deprecated in a future Kubernetes release.
 
+### Node Affinity
+
+A PV can specify [node affinity](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volumenodeaffinity-v1-core) to define constraints that limit what nodes this volume can be accessed from. Only the nodes that satifsy the constraints can access the volume.
+
+Node affinity is conceptually similar to NodeSelector.
```

Comment:
Would it be simpler to add a link to the node affinity/node selector page?

Reply:
Added a link to NodeSelector page